constantinCurtis/src/main/java/data/collection/task/Main.java

```diff
@@ -8,8 +8,7 @@
 import java.util.Map;
 import java.util.logging.Logger;
 
-import static data.collection.task.classes.Order.filterOrderByItemType;
-import static data.collection.task.classes.User.filterUsersByCity;
+import static data.collection.task.classes.AbstractUser.filterUsersByCity;
 
 public class Main {
```

Comment:
if you have a Main class where you would like to check your work, please format it properly and don't leave unused/unassigned statements

Reply:
changed, thank you

Reply:
ALT+ENTER sounds good, but in this case you can call the method by AbstractUser.filterUsersByCity. Let's be more genereic, import the class, then use the class methods.

Reply:
Thank you, resolved